bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -1358,7 +1358,7 @@ CGINS-FLORTD-01487,,Sensor,0,3-Wavelength Fluorometer: FLORT Series D,FLORT SERI
 CGINS-FLORTD-01488,,Sensor,0,3-Wavelength Fluorometer: FLORT Series D,FLORT SERIES D (ECO TRIPLET-W),WET Labs,ECO Triplet-w,1488,,20160812,,,
 CGINS-FLORTJ-01084,N00593,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-001,WET Labs,ECO Triplet-w,BBFL2W-1084,,20130630,13013.52,,Oregon State University
 CGINS-FLORTJ-01156,OL000612,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-006,WET Labs,ECO Triplet-w,BBFL2W-1156,,20140929,13013.52,also N00601,Oregon State University
-CGINS-FLORTJ-01205,N00561,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-005,WET Labs,ECO Triplet-w,1205,,20140801,13013.52,,
+CGINS-FLORTJ-01205,N00561,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-005,WET Labs,ECO Triplet-w,BBFL2W-1205,,20140801,13013.52,,
 CGINS-FLORTJ-01206,N00585,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-002,WET Labs,ECO Triplet-w,BBFL2W-1206,,20140630,13013.52,,Oregon State University
 CGINS-FLORTJ-01207,N00577,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-003,WET Labs,ECO Triplet-w,BBFL2W-1207,,20140616,13013.52,,Oregon State University
 CGINS-FLORTJ-01208,N00569,Sensor,0,3-Wavelength Fluorometer: FLORT Series J,FLORT for WLP-004,WET Labs,ECO Triplet-w,BBFL2W-1208,,20140801,13013.52,,

```

Comment:
Looks good.  Side note - at some point we need to bring the 15 or so FLORDMs and probably these FLORTJs in line w/ the majority of the other FLORD/Ts and remove the model designation from the S/Ns (exa BBFL2W).

Reply:
Looks good.